tests/20-rdfc-verify.js

```diff
@@ -1,84 +1,183 @@
 /*!
- * Copyright 2023 Digital Bazaar, Inc.
+ * Copyright 2024 Digital Bazaar, Inc.
```

Comment:
This was originally a new file but I suppose it should retain the original notice now that it’s the same

Reply:
just expanded the copyright 2023-2024